src/main/java/seedu/student/storage/JsonAdaptedAppointment.java

```diff
@@ -19,37 +20,38 @@
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Appointment's %s field is missing!";
 
     private final String matriculationNumber;
-    private final String date;
-    private final String startTime;
+    private final LocalDate date;
+    private final LocalTime startTime;
```

Comment:
Is there a reason you use `LocalDate` and `LocalTime` here instead of `String`?

Reply:
Since date and startTime are supposed to be of type `LocalDate` and `LocalTime` respectively, it would be a better idea to just store them as their own type instead of `String`.